pedsnetdcc/prepdb.py

```diff
@@ -0,0 +1,240 @@
+import logging
+import re
+import psycopg2
+
+logger = logging.getLogger(__name__)
+
+
+def _conn_str_with_database(conn_str, dbname):
```

Comment:
I'm guessing these first four functions will move to `utils.py`, but we can wait until they are actually needed by other modules so as not to be moving things back and forth needlessly.